samples/ftd_configuration/anyconnect_package_file.yaml

```diff
@@ -0,0 +1,45 @@
+- hosts: all
+  connection: httpapi
+  gather_facts: no
+  vars_files:
+  - variables.yaml
+  tasks:
+  - name: Check Anyconnect Package File presense
```

Comment:
Let's use `AnyConnect` case to match the product name...

Reply:
Done